Classes/Domain/PrototypeDetails/PrototypeDetailsInterface.php

```diff
@@ -53,6 +54,11 @@ public function getAnatomy(): Anatomy;
      */
     public function getProps(): PropsCollectionInterface;
 
+    /**
+     * @return UseCaseCollection
+     */
+    public function getUseCases(): UseCaseCollection;
```

Comment:
missing in interface

Reply:
Only if there is a need to add another collection implementation

Reply:
my phpstorm was just confused sometimes ^^